index.js

```diff
@@ -23,6 +23,17 @@ Vector.fromObject = function (obj) {
   return new Vector(obj.x, obj.y);
 };
 
+Vector.fromString = function (str) {
+  // e.g.  'x:100, y:200'
```

Comment:
What would you say about accepting `x: 100, y: 200` as correct string as well? And what's gonna happen if you `fromString('unparsable thing here')`? I'd say throwing a clear error would be nice and might save you one day.